### PR TITLE
Added the About page and tab to the PageNav

### PR DESCRIPTION
**Pull request by SlamBamActionman**
Does what it says on the tin
image

### CODE REVIEW

**Comment by dallonf** [comment]
Onoz lint errors! 😮

File: src/Routes.tsx [hunk 1]
```diff
@@ -14,6 +15,12 @@ export const Routes: React.FC = () => (
     <Route exact={true} path="/">
       <Home />
     </Route>
+    <Route exact={true} path="/about">
+      <About />
+    </Route>
+    <Route exact={true} path="/login">
```
Comment (Willdotwhite) [inline src/Routes.tsx:21]: I removed this in my Interstitial Login Page change - is this coming back intentionally, or has git snuck this in under the radar?

Comment (SlamBamActionman) [reply src/Routes.tsx:21]: Heck, I am sure I deleted that when I solved the merge conflicts. I'll fix it

**Comment by Willdotwhite** [comment]
I'm not sure if React/HTML does any particular magic around text in `<div>` elements, but I think strictly they should be in `<p>` elements shouldn't they?

Not so concerned to stop approval though, happy to fix that forward

**Review by Willdotwhite — APPROVED** [review]

**Pull request merged**